tests/unit_tests/test_lattice_discretization.py

```diff
@@ -0,0 +1,147 @@
+from math import sqrt
+import xml.etree.ElementTree as ET
+
+import openmc
+import pytest
+
+from tests.unit_tests.test_lattice import zr, pincell1, pincell2, rlat2
+
+
+def test_discretization_degenerate(rlat2):
+
```

Comment:
Remove the unnecessary whitespace after the function definition here and elsewhere in the tests functions you've added

Reply:
done